### PR TITLE
Update animal.py

### PR DESCRIPTION
**Pull request by dupontjean**
added big cats

### CODE REVIEW

**Review by HTML-CODDER — COMMENTED** [review]
Update animal.py #13

File: animal/animal.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
 dogapi = "https://dog.ceo/api/breeds/image/random"
 foxapi = "http://wohlsoft.ru/images/foxybot/randomfox.php"
 pugapi = "http://pugme.herokuapp.com/random"
+bigcat = "http://randombig.cat/roar.json"
```
Comment (skeith) [inline animal/animal.py:16]: change this to `roarapi` for consistency

File: animal/animal.py [hunk 2]
```diff
@@ -27,6 +28,7 @@ def __init__(self, bot):
         self.dogapi = dogapi
         self.foxapi = foxapi
         self.pugapi = pugapi
+        self.bigcat = bigcat
```
Comment (skeith) [inline animal/animal.py:31]: which means, this and subsequent occurrence should be changed to `self.roarapi`

**Review by skeith — CHANGES_REQUESTED** [review]
Change the variable name for consistency

**Pull request closed without merging**